src/preferences/dialog/dlgprefcontrolsdlg.ui

```diff
@@ -413,6 +413,33 @@
           </attribute>
          </widget>
         </item>
+        <item row="3" column="0">
+         <widget class="QLabel" name="labelKeyunlockMode">
+          <property name="text">
+           <string>Keyunlock mode</string>
+          </property>
+         </widget>
+        </item>
+        <item row="3" column="1">
+         <widget class="QRadioButton" name="radioButtonCalculateNewKey">
+          <property name="text">
+           <string>Adapt key to current tempo</string>
```

Comment:
Actually the first time I heard of 'linear pitch' was in the source code. It just has no meaning to me since linear somehow refers to a motion, like exponential, not to a state.

> "Reset key" and "Keep key"

Perfect!